exec/RenderHtml.hs

```diff
@@ -75,32 +75,38 @@ pages f =
     ]
 
 
+-- | We write 'ToHtml' for pages that contain no forms, and 'FormPageView' for pages that do.  In
+-- this module, we need to render both into html for viewing only, and 'ToHtml'' is introduced for
+-- this.  We can instantiate wrapper types for 'ToHtml' instances and 'FormPageView', resp., and get
+-- a uniform way of rendering html for either.
 class ToHtml' p where
     toHtml' :: Monad m => p -> HtmlT m ()
 
+-- | A wrapper type to make all 'ToHtml' instances 'ToHtml'' instances.
```

Comment:
Here @ToHtml'@ would really help --- I thought these are the same. BTW, if anybody has a good idea, a name without a prime would be better (though I see how @ToHtml'@ makes sense).